src/test/java/FirstTests.java

```diff
@@ -1,10 +1,12 @@
+import org.testng.annotations.Listeners;
 import org.testng.annotations.Test;
 
 import java.security.SecureRandom;
 import java.util.Random;
 
 import static org.testng.AssertJUnit.assertEquals;
 
+@Listeners(Listener.class)
```

Comment:
Ten listener jest tu potrzebny wyłącznie wtedy kiedy chcesz aby zadziałał przy odpalaniu testów bezpośrednio z klasy (np przez task gradle z poprzedniego zadania). Jeżeli chcesz odpalać testy wyłącznie poprzez odpalanie suitów to można go usunąć i zostawić listener w xmlu z odpowiednim suitem.

Reply:
Tak, wiem. Został tutaj ponieważ coś testowałam jak działa i zapomniałam usunąć.